models/jules/R/write.config.JULES.R

```diff
@@ -579,12 +587,51 @@ write.config.JULES <- function(defaults, trait.values, settings, run.id) {
 
     ## write namelist
     writeLines(ic.text, con = ic.file)
-
+    
+    if (is.null(settings$run$inputs$poolinitcond$path)){
     ## also load and parse IC dat file
-    ic.dat <- file.path(local.rundir, "initial_conditions.dat")
-    ic.text <- readLines(con = ic.dat, n = -1)
-    ic.text[2] <- paste(ic.text[2]," 5.0 5.0 0.5 0.5 0.5 0.2 0.2 0.2 0.2 0.2 0.0 0.0 0.0 0.0")
-    writeLines(ic.text, con = ic.dat)
+    ## also load and parse IC dat file
+      ic.dat <- file.path(local.rundir, "initial_conditions.dat")
+      ic.text <- readLines(con = ic.dat, n = -1)
+      ic.text[2] <- paste(ic.text[2]," 5.0 5.0 0.5 0.5 0.5 0.2 0.2 0.2 0.2 0.2 0.0 0.0 0.0 0.0")
+      writeLines(ic.text, con = ic.dat)
+    
+    } else if (!is.null(settings$run$inputs$poolinitcond$path)){
```

Comment:
The ```if``` here in the ```else if``` is completely redundant with the previous ```if``` (if the last check what if the path was null and was false, then you already know that this check has to be true).  A better check would be ```file.exists```